src/App.js

```diff
@@ -1,25 +1,78 @@
-import logo from './logo.svg';
+import React, {useEffect, useState} from 'react';
```

Comment:
```suggestion
import {useEffect, useState} from 'react';
```

Este no es del todo necesario que lo pongas, salvo que lo utilices mas adelante, en principio no te hace falta